### DISCUSSION TITLE
[Native Filters] Add Time Range filter as available in Query History

### DISCUSSION THREAD

**Discussion by kamalkeshavani-aiinside** [apache/superset#22000] [Ideas]
Most of end users don't like the highly complicated(also useful) time range native filter currently available.

<img width="719" alt="image" src="https://user-images.githubusercontent.com/74634977/199409709-2e3d1f7b-09a0-41f1-b827-5be98db9b203.png">

So, it would be great if the below Time Range filter(already available in Query History page) is also made available as native filter.

<img width="892" alt="image" src="https://user-images.githubusercontent.com/74634977/199409593-a2b84ccc-419f-4325-a60f-c342848a6d5c.png">

**Comment by sfirke** [comment]
+1 to this.  The current filtering is powerful but too complex for many of my users.  I work in government and expose Superset reports to members of the public with all levels of technical comfort.  This calendar picker would be a great addition to the list in the first screenshot, with a simple name like "Date Range".

It overlaps some with "Custom" which currently can do this, but is harder to use.  Probably keep Custom as-is?  I'd like the Time Range picker to have the choice of "Now" for end date or start date.

**Reply by kamalkeshavani-aiinside** [reply]
'Now' button is a great suggestion.
